### PR TITLE
Adding a11y tests for on demand radio episode pages

### LINKED ISSUE #6239: On Demand Radio Page a11y tests
Run automated a11y tests for a sample on demand radio pages. This requires adding the on demand episode pages to the Cypress config, and marking 1-2 services as `smoke: true`

This should also include adding exceptions to the bbcA11y exceptions for on demand radio episodes. See https://github.com/bbc/simorgh/pull/6189

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #6239

**Overall change:**
Adding a11y tests for on demand radio episode pages

**Code changes:**
- add onDemandRadioEpisode page type
- add exceptions for SMP iframe & RTL scrollable navigation

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: a11y.js [hunk 1]
```diff
@@ -9,21 +9,20 @@ const services = require('./cypress/support/config/services');
...
-const getUrls = (pageType) =>
+const getUrls = pageType =>
   Object.keys(services)
     .map(getPageTypes)
-    .map((pageTypes) => pathOr(null, [pageType], pageTypes))
+    .map(pageTypes => pathOr(null, [pageType], pageTypes))
     .map(getSmokePaths)
-    .filter((page) => !!page)
-    .map((url) => `${baseUrl}${url}`);
+    .filter(page => !!page)
```
Comment (ryanmccombe) [inline a11y.js]:
```suggestion
    .filter(Boolean)
```

File: a11y.js [hunk 2]
```diff
@@ -9,21 +9,20 @@ const services = require('./cypress/support/config/services');
 const pageWidths = [360];
 const baseUrl = 'http://localhost:7080';
 
-const getPageTypes = (service) =>
-  pathOr(null, [service, 'pageTypes'], services);
+const getPageTypes = service => pathOr(null, [service, 'pageTypes'], services);
```
Comment (ryanmccombe) [inline a11y.js]:
```suggestion
const getPageTypes = service => path([service, 'pageTypes'], services);
```

File: a11y.js [hunk 3]
```diff
@@ -9,21 +9,20 @@ const services = require('./cypress/support/config/services');
...
   const { environments, smoke } = config;
   return smoke && environments ? environments.local.paths : null;
 };
 
-const getUrls = (pageType) =>
+const getUrls = pageType =>
   Object.keys(services)
     .map(getPageTypes)
-    .map((pageTypes) => pathOr(null, [pageType], pageTypes))
+    .map(pageTypes => pathOr(null, [pageType], pageTypes))
```
Comment (ryanmccombe) [inline a11y.js]:
```suggestion
    .map(path([pageType]))
```

File: a11y.js (hunk 2 above)
Comment (ryanmccombe) [reply a11y.js]: and line 1: `const path = require('ramda/src/path');` (out of range for github suggestion 😢 )

**Review by ryanmccombe — APPROVED** [review]
👍

File: a11y.js (hunk 1 above)
Comment (karinathomasbbc) [reply a11y.js]: I think I can actually remove it altogether.

Comment (karinathomasbbc) [reply a11y.js]: Mmm nope, no idea why but will use `filter(Boolean)` instead

**Review by pharingee — APPROVED** [review]
Nice refactor!

**Pull request merged**